Cargo.toml

```diff
@@ -334,6 +339,7 @@ reqwest = { version = "0.12.7", default-features = false, features = [
   "zstd",
   "json",
 ] }
+rlp = "0.5.2"
```

Comment:
Can you please move this standalone entry to use the new `workspace` version?
https://github.com/alpenlabs/alpen/blob/e2a4138a3f18e375a7f26b6a7f24572f12cb93ae/crates/mpt/Cargo.toml#L17